docs/ref/reference.rst

```diff
@@ -92,6 +92,7 @@ commands are dealing with the monitor:
         --pgport      PostgreSQL's port number
         --nodename    hostname by which postgres is reachable
         --auth        authentication method for connections from data nodes
+        --no-pg-hba   Do not allow pg_autoctl to manage pg_hba.conf
```

Comment:
I think I would like `--skip-hba` or `--skip-pg-hba` better. The rationale is to avoid negative commands and then the infamous *if not notThis then that* that makes every debugging session harder than necessary.